source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Replacing fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+Since version 4.3 Rsyslog is used to gather metrics from oVirt hosts (replacing fluentd).
```

Comment:
@eslutsky In oVirt 4.3 Fluentd was replaced by Rsyslog to gather metrics and logs from the oVirt hosts and engine.